app/views/foreman_tasks/tasks/dashboard/_latest_tasks_in_error_warning.html.erb

```diff
@@ -1,15 +1,17 @@
 <h4><%= link_to _("Latest Warning/Error Tasks"), foreman_tasks_tasks_path(:order=>'started_at DESC') %></h4>
-<table class="table table-striped">
+<table class="table table-striped table-fixed">
   <tr>
-    <th><%= _("Name") %></th>
-    <th><%= _("State") %></th>
-    <th><%= _("Result") %></th>
+    <th class="col-md-5"><%= _("Name") %></th>
+    <th class="col-md-2"><%= _("State") %></th>
+    <th class="col-md-2"><%= _("Result") %></th>
+    <th class="col-md-3"><%= _("Ended at") %></th>
```

Comment:
Would it make sense to drop the 'at' from the header or reword it some other way? `Ended at: 5 minutes ago` feels weird, but maybe it's just me.